Bio/PDB/Polypeptide.py

```diff
@@ -197,8 +197,11 @@ def get_ca_list(self):
         """
         ca_list = []
         for res in self:
-            ca = res["CA"]
-            ca_list.append(ca)
+            try:
```

Comment:
Nevermind, my bad :) Thought `res['CA']` was accessing `res.child_dict` and there you'd have a `.get` method.